examples/warehouse/service/dns-resolver-example-com/unbound.yaml

```diff
@@ -0,0 +1,15 @@
+# The unbound DNS resolver <https://unbound.net/> has a simple
+# key-value configuration structure with top-level categories and
+# underlying key-value items.  The YAML structure mirrors this.
+#
+# Value lists are represented as repeted key-values.  Strings in
+# unbound config are double-quoted, so any quoting in the YAML
+# structure should use single-quoutes to prevent confusion.
```

Comment:
"quotes"

Also, the entire sentence structure feels weird. If the intention is that the strings read should include the double quotes (which it looks like, from `unbound_config` below), I'd prefer something like "The Unbound configuration format requires string values to be surrounded by double quotes. To preserve them through the YAML parser, surround the entire value with single quotes here".